src/communication/serverMessages.ts

```diff
@@ -25,16 +26,18 @@ export interface Data {
 
 export interface CommandData {
     Command: string;
-    JsonMessage: string;
+    Data: any;
 }
 
 export let convertToWebsocketInputFormat = (data: Data): string => {
```

Comment:
Not sure it's the right place.
convertToWebsocketInputFormat is not a server message